<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtngImageWidget</class>
 <widget class="QWidget" name="PtngImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PtngImageWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMain">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="labelBanner">
     <property name="text">
      <string>&lt;No file selected&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>345</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonZoomIn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Zoom in</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonZoomOut">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Zoom out</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonOrig">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Scale to original</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>530</width>
        <height>542</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutScrollArea">
       <item row="1" column="0">
        <widget class="QLabel" name="labelImage">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/icons.qrc">:/goldings_gym_core_00_white.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
